docs/docs/use-cases/flux.md

```diff
@@ -0,0 +1,189 @@
+---
+comments: true
+---
+
+# Deploying Applications with Flux
+
+Flux is a tool for keeping Kubernetes clusters in sync with sources
+of configuration (like Git repositories), and automating updates to
+configuration when there is new code to deploy.
+
+This section shows an already existing Flux use case of running
+[pre and post-deployment jobs with Flux](https://fluxcd.io/flux/use-cases/running-jobs/)
+and how Keptn makes it simpler and and more straight-forward.
+
+## High-level structure of the git repository
+
+Since Flux uses a GitOps approach for continuous delivery, the git
+repository structure, for our use case, will look like the following:
```

Comment:
```suggestion
Since Flux uses a GitOps approach for continuous delivery, the git
repository structure for our use case looks like the following:
```